Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/CreateAndQueryChangeRequestV2/task.json

```diff
@@ -354,7 +354,35 @@
+          },
+          "ExecutionOptions": {
+            "OutputVariables": "{\"CHANGE_REQUEST_NUMBER\" :  \"jsonpath('$.result[0].number')[0]\", \"CHANGE_CORRELATION_ID\" :  \"jsonpath('$.result[0].correlation_id')[0]\", \"CHANGE_SYSTEM_ID\" :  \"jsonpath('$.result[0].sys_id')[0]\"}",
+            "SkipSectionExpression": "or(eq(eq('$(system.hosttype)', 'checks'), false), or(eq(eq(taskInputs['changeRequestAction'], 'createNew'), false), and(eq(taskInputs['changeRequestAction'], 'createNew'), eq(isNullOrEmpty(variables['CHANGE_SYSTEM_ID']), false))))"
+          }
+        },
+        {
+          "RequestInputs": {
+            "EndpointId": "$(ServiceNowConnection)",
+            "EndpointUrl": "$(endpoint.url)/api/now/import/x_mioms_azpipeline_change_request_import",
+            "Method": "POST",
+            "Body": "{  \"u_correlation_id\": \"{{#newGuid}}{{/newGuid}}\",\"u_type\": \"$(changeType)\",\"u_x_mioms_azpi_eline_metadata\": \"Collection URI: $(system.collectionUri) \\r\\n Project: $(system.teamProject) \\r\\n StageName: $(system.stageName) \\r\\n RequestedFor: $(build.requestedFor)\", \"u_short_description\": \"$(shortdescription)\"{{#equals changeType 'Normal' 1}}{{#if description}}, \"u_description\": \"$(description)\"{{/if}}{{#if category}}, \"u_category\": \"$(category)\"{{/if}}{{#if priority}}, \"u_priority\": \"$(priority)\"{{/if}}{{#if risk}}, \"u_risk\": \"$(risk)\"{{/if}}{{#if impact}}, \"u_impact\": \"$(impact)\"{{/if}}{{#if configurationitem}}, \"u_cmdb_ci\": \"$(configurationitem)\" {{/if}}{{#if assignmentgroup}}, \"u_assignment_group\": \"$(assignmentgroup)\"{{/if}}{{/equals}}, \"u_requested_by_date\": \"{{toDateTimeFormat '$(system.checksStartTime)' 'dd-MM-yyyy HH:mm:ss UTC'}}\", \"u_start_date\": \"{{#if schedulestarttime}}{{toDateTimeFormat '$(schedulestarttime)' 'dd-MM-yyyy HH:mm:ss UTC'}}{{/if}}\",  \"u_end_date\": \"{{#if scheduleendtime}}{{toDateTimeFormat '$(scheduleendtime)' 'dd-MM-yyyy HH:mm:ss UTC'}}{{/if}}\" {{#if otherParameters}}{{toCommaSeparatedKeyValueList otherParameters true}}{{/if}}{{#equals changeType 'Standard' 1}}{{#if standardChangeTemplate}} , \"template_id\": \"$(standardChangeTemplate)\" {{/if}}{{/equals}}, \"u_stage_id\": \"$(system.stageId)\", \"u_build_id\": \"$(build.buildId)\", \"u_stage_attempt\": \"$(checks.stageAttempt)\" }",
+            "Headers": "{\"Content-Type\":\"application/json\", \"Accept\":\"application/json\"}",
+            "WaitForCompletion": "false",
+            "Expression": "eq(jsonpath('$.result[0].status')[0], 'inserted')"
+          },
+          "ExecutionOptions": {
+            "OutputVariables": "{\"CHANGE_SYSTEM_ID\" :  \"jsonpath('$.result[0].sys_id')[0]\"}",
+            "SkipSectionExpression": "or(eq(eq('$(system.hosttype)', 'checks'), false), or(eq(eq(taskInputs['changeRequestAction'], 'createNew'), false), and(eq(taskInputs['changeRequestAction'], 'createNew'), eq(isNullOrEmpty(variables['CHANGE_SYSTEM_ID']), false))))"
```

Comment:
One condition seems to be redundant 
i believe the condition is like (1 || (2 || (3 && 4))
where 2 is eq(eq(taskInputs['changeRequestAction'], 'createNew'), false)
and 3 & 4 are eq(taskInputs['changeRequestAction'], 'createNew'), 
 and eq(isNullOrEmpty(variables['CHANGE_SYSTEM_ID']), false))
respectively 

so it will go to block of (3 && 4 ) only when 2 is false , which means taskInputs['changeRequestAction'] is 'createNew'
hence 3 automatically becomes true , isn't it ?

Reply:
removed unnecessary conditions